app/models/course.rb

```diff
@@ -553,6 +541,18 @@ def assignable_subjects
 
 private
 
+  def add_site!(site:)
+    is_course_new = ucas_status == :new
```

Comment:
This temporary variable is probably not adding anything, and also a bit odd as `remove_site!` doesn't bother with it. Happy to worry about this later if you prefer.